test/net/sourceforge/kolmafia/KoLCharacterTest.java

```diff
@@ -297,6 +297,35 @@ public void familiarsWithoutGsAreStillOwnedInGLover() {
     var cleanups =
         new Cleanups(withFamiliarInTerrarium(FamiliarPool.MOSQUITO), withPath(Path.GLOVER));
 
+    try (cleanups) {
+      var fam = KoLCharacter.ownedFamiliar("mosquito");
+      assertThat(fam.isPresent(), is(true));
+      assertThat(fam.get().getId(), is(FamiliarPool.MOSQUITO));
+    }
+  }
+
+  @Test
+  public void graftedFamiliarsArentUsableInZootomist() {
+    var cleanups =
+        new Cleanups(
+            withFamiliarInTerrarium(FamiliarPool.MOSQUITO),
+            withPath(Path.Z_IS_FOR_ZOOTOMIST),
+            withProperty("zootGraftedButtCheekLeftFamiliar", FamiliarPool.MOSQUITO));
+
+    try (cleanups) {
+      var fam = KoLCharacter.usableFamiliar("mosquito");
+      assertThat(fam, nullValue());
+    }
+  }
+
+  @Test
+  public void graftedFamiliarsAreOwnedInZootomist() {
```

Comment:
I would rather these two tests be combined to make it obvious they have the same setup; I think this makes clearer the intent that grafted familiars are owned but unusable.